### DISCUSSION TITLE
能不能添加一个挂载到本地的功能

### DISCUSSION THREAD

**Discussion by DifferentSparkler** [AlistGo/alist#978] [Ideas]
https://soft.kermsite.com/p/clouddrive/
![挂载1](https://user-images.githubusercontent.com/38945550/164171335-f0dcbc15-3b40-4099-a843-3ef0c0700ef3.png)
![挂载](https://user-images.githubusercontent.com/38945550/164171358-ee16811a-2e36-4c88-a8a6-38ed369b0524.png)

**Comment by PANDAZZ1108** [comment]
你可以用这个挂载本地的。

**Reply by DifferentSparkler** [reply]
> 你可以用这个挂载本地的。

怎么操作啊

**Reply by PANDAZZ1108** [reply]
alist，本身有webdav功能，然后你在用你发的截图那个软件挂载到本地就行了。但是得有一台装alist的服务器。

**Reply by DifferentSparkler** [reply]
> alist，本身有webdav功能，然后你在用你发的截图那个软件挂载到本地就行了。但是得有一台装alist的服务器。
你没有明白我的意思  我的意思是这个软件直接添加下这个功能

**Reply by dew556** [reply]
![图片](https://user-images.githubusercontent.com/51583204/166476353-92f06cf4-afe8-4fdc-ae58-70aa2a270fd1.png)

**Reply by DifferentSparkler** [reply]
我说的是alist这个项目添加一个类似clouddrive的挂载功能 这样就不需要借用其它的工具实现挂载到本地的功能
